### ISSUE TITLE
Formatting issue for compiler-tools.rst

### ISSUE THREAD

**Issue by brettcannon** [conda/conda-docs#513]
If you look at https://github.com/conda/conda-docs/blob/master/docs/source/user-guide/tasks/build-packages/compiler-tools.rst you will notice that there's a bare URL in the documentation (search for "L79"). My suspicion is that was either meant to be a link or a code snippet.

**Comment by mingwandroid** [comment]
Yeah, linking to a line number in a file that changes is not the best. Currently it should be: https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L90-L91

But best to copy those lines into the docs.

**Comment by brettcannon** [comment]
You can also make the link permanent to a specific revision so it won't necessarily become invalid going forward (but obviously if the setting itself changed then the link might be pointing to something that is no longer valid).
